test/integration/instance_stats_test.go

```diff
@@ -6,8 +6,8 @@ import (
 	"time"
 )
 
-func TestGetInstanceStats(t *testing.T) {
-	client, instance, teardown, err := setupInstance(t, "fixtures/TestGetInstanceStats")
+func TestInstanceStats_Get(t *testing.T) {
```

Comment:
This test seems to be timing out fairly consistently when run against against existing fixtures. It might be a good idea to just `t.Skip()` here to get the PR workflow working more consistently.